project3/index1.html

```diff
@@ -25,254 +25,340 @@ <h2>With great power there must also come-great responsibility</h2>
-                    <path d="M124.6050034,103.9775009l-6.8736038-1.0512009l-3.1103058-6.6259995C113.9618988,94.8964996,112.5507965,94,111,94  c-1.5508041,0-2.9619064,0.8964996-3.6211014,2.3003006l-3.1102982,6.6259995l-6.8736038,1.0512009  c-1.4818954,0.2266006-2.7123947,1.2631989-3.1884995,2.6846008c-0.4754944,1.4218979-0.1161957,2.9901962,0.9307022,4.0629959  l5.0903015,5.2168045l-1.209404,7.4141006c-0.2465973,1.5116959,0.3916016,3.0317001,1.6439972,3.9139938  C101.3500977,127.7544022,102.1567001,128,102.966301,128c0.6645966,0,1.3305969-0.1654968,1.9350967-0.4994965L111,124.1284027  l6.0985947,3.3721008C117.7030945,127.8345032,118.3695984,128,119.033699,128c0.8096008,0,1.6161957-0.2455978,2.3041992-0.7300034  c1.2523956-0.8822937,1.8906021-2.402298,1.6439972-3.9139938l-1.2093964-7.4141006l5.0903015-5.2168045  c1.0468979-1.0727997,1.4061966-2.641098,0.9306946-4.0629959  C127.3173981,105.2406998,126.0868988,104.2041016,124.6050034,103.9775009z M117.4994965,114.5932999L119.0341949,124  L111,119.5576019L102.9657974,124l1.5346985-9.4067001L98,107.9316025l8.9828949-1.3740005L111,98l4.0170975,8.5576019  L124,107.9316025L117.4994965,114.5932999z" style="fill: rgb(255, 255, 255);"/>
-                    <path d="M113.8798981,4.22998c0.1400986,0,0.2798004-0.02002,0.419899-0.0502901  c1.080101-0.22998,1.7602997-1.2998099,1.5298004-2.37989c-0.2300034-1.0800731-1.2997971-1.7597611-2.3798981-1.52978  c-1.080101,0.22998-1.7598038,1.2998-1.5298004,2.37988C112.1195984,3.5898399,112.9496994,4.22998,113.8798981,4.22998z" style="fill: rgb(255, 255, 255);"/>
-                    <path d="M122.3295975,6.2597699c0.580101,0,1.1504059-0.25,1.5503006-0.72998  c0.6996994-0.85987,0.5698013-2.1201198-0.280304-2.8198299c-0.8598938-0.7001901-2.1195908-0.5703099-2.8197937,0.2797899  c-0.7002029,0.8598602-0.5698013,2.12011,0.2802963,2.8203101C121.4398956,6.1098599,121.8798981,6.2597699,122.3295975,6.2597699z" style="fill: rgb(255, 255, 255);"/>
-                    <path d="M113.2798004,23.6898994c0.1498947,0.0401001,0.3099976,0.0498009,0.4598999,0.0498009  c0.9101944,0,1.7299957-0.6201,1.9399948-1.5498009c0.25-1.0702991-0.419899-2.1498985-1.4897995-2.3998985  c-1.0800018-0.25-2.1600952,0.4199982-2.3998947,1.5C111.5400009,22.3696003,112.2099991,23.4398994,113.2798004,23.6898994z" style="fill: rgb(255, 255, 255);"/>
-                    <path d="M122.219696,21.8295994c0.4400024,0,0.8798981-0.1396999,1.240303-0.4296989  c0.8695984-0.6899014,1.009697-1.950201,0.3198013-2.8101006c-0.6802063-0.8701-1.9400024-1.0102005-2.8101044-0.3197994  c-0.8598022,0.6796989-1,1.9399986-0.3198013,2.8095989C121.0497971,21.5697994,121.6298981,21.8295994,122.219696,21.8295994z" style="fill: rgb(255, 255, 255);"/>
-                    <circle cx="126" cy="12" r="2" style="fill: rgb(255, 255, 255);"/>
-                    <path d="M106.9599991,18.2798004c0.2900009,0,0.5800018-0.0601006,0.8597946-0.1900005  c1-0.4701996,1.4199066-1.6698
```

Comment:
Since this `div` wraps around a `label` which already wraps around all the other elements, it's probably not necessary; you can add your classname and styles directly to the `label` and remove the need for the `div`.